### ISSUE TITLE
Using :external twice in the same selector only transforms the first one

### ISSUE THREAD

**Issue by chielkunkels** [tivac/modular-css#342]
Hey Pat,

Found a small bug with `:external`:

Doing something like
```css
:external(content from './page.css') :external(container from './summary.css') {
	opacity: .5;
}
```
 Comes out as
```css
.mc10fec9bb_content :external(container from './summary.css') {
  opacity: .5;
}
```

If I nest the thing (using `postcss-nested`)
```css
:external(content from './page.css') {
	:external(container from './summary.css') {
		opacity: .5;
	}
}
```
It correctly comes out as
```css
.mc10fec9bb_content .mca9609b55_container {
  opacity: .5;
}
```

Not a huge issue since it's easily worked around, bug figured you might wanna know.

As always, thanks for the amazing package, using it all my projects, at work and personal.

Cheers,
Chiel

**Comment by tivac** [comment]
Blergh, sorry! Will take a look when I can which is probably late tonight or this weekend.

Yet more proof that I haven't used `:external` enough in my own code... 😒

**Comment by chielkunkels** [comment]
No need to apologise, like I said I worked around it for now, so for me personally no rush at all!

I am often not sure if I should use `:external` or `composes` to be honest.

**Comment by tivac** [comment]
I use `composes` whenever possible, I see `:external` as an escape-hatch for when you just have to do something you maybe shouldn't.

But that's just my personal opinion, I've got no desire to try and enforce that via code.

**Comment by chielkunkels** [comment]
Yeah makes sense, I use `:external` if I don't actually have anything to add to the selectors, but I should definitely use `composes` more. Regardless, good to make sure as it works on the tin I suppose :)

**Comment by tivac** [comment]
This is fixed in `6.0.2`

**Comment by chielkunkels** [comment]
Awesome, thanks mate!
